### PR TITLE
LL-6883 - Add platform deeplink and handeling of parameters

### PR DESCRIPTION
**Pull request by klambert-ledger**
## 🦒 Context (issues, jira)
[LL-6883](https://ledgerhq.atlassian.net/browse/LL-6883)

## 💻  Description / Demo (image or video)
Adding new deeplinks:

`ledgerlive://discover` Should open the catalog

`ledgerlive://discover/:APPID?param1=a&param2=b&param3=...`  should point to the app page, forwarding any extra parameter to the live app.

### Video
<details>
    <summary> Mac</summary>

https://user-images.githubusercontent.com/86959861/132031259-76b677ac-8373-4167-b7df-5d1bd9cc1801.mov

</details>

## 🖤  Expectations to reach

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

### CODE REVIEW

File: src/renderer/hooks/useDeeplinking.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ export function useDeepLinkHandler() {
     (event: any, deeplink: string) => {
       const { pathname, searchParams } = new URL(deeplink);
       const query = Object.fromEntries(searchParams);
-      const url = pathname.replace(/(^\/+|\/+$)/g, "");
+      const fullUrl = pathname.replace(/(^\/+|\/+$)/g, "");
+      const [url, path] = fullUrl.split("/");
```
Comment (nparigi-ledger) [inline src/renderer/hooks/useDeeplinking.js:57]: We did pretty much the same with my PR : 
https://github.com/LedgerHQ/ledger-live-desktop/pull/4148/files#diff-df55ff13a3c744695e8cb8ba3fa2d513e90a10b631704f7be6a3faaffa76247cR57
Risk of merging issues :sweat_smile:

**Review by nparigi-ledger — DISMISSED** [review]

File: src/renderer/hooks/useDeeplinking.js (hunk 1 above)
Comment (lambertkevin) [reply src/renderer/hooks/useDeeplinking.js:57]: Thx will take care of the rebase when yours is merged 👍

Comment (klambert-ledger) [reply src/renderer/hooks/useDeeplinking.js:57]: Thx will take care of the rebase when yours is merged 👍

Comment (gre) [reply src/renderer/hooks/useDeeplinking.js:57]: the other pr is merged

**Review by LFBarreto — APPROVED** [review]
:link:

**Pull request merged**